pkg/wal/model.go

```diff
@@ -0,0 +1,31 @@
+package wal
+
+import "gopkg.in/yaml.v2"
+
+// All the serializable model for the WAL.
+
+type Entry struct {
+	Token   string `json:"token" yaml:"token"`
+	Payload string `json:"payload" yaml:"payload"`
+}
+
+const tokenGeneratorPath = "WALTokenGeneratorPath"
+const maxConcurrency = 1024
+
+func NewEntry(token string, payload string) *Entry {
+	return &Entry{
+		Token:   token,
+		Payload: payload,
+	}
+}
+
+func Unmarshal(b []byte) (*Entry, error) {
```

Comment:
I think we should rather implement here the YAML unmarshaller interface: https://github.com/go-yaml/yaml/blob/v2.2.4/yaml.go#L32

Reply:
Makes sense. Will do it.